helm-charts/aerospike-kubernetes-operator/values.yaml

```diff
@@ -20,12 +20,16 @@ rbac:
 
 ## Ports
 healthPort: 8081
-metricsPort: 8080
+metricsPort: 8443
 
 ## Create certificates
 certs:
-  create: true
-  webhookServerCertSecretName: "webhook-server-cert"
+  webhook:
+    create: true
```

Comment:
can user set this to false?

Reply:
Webhook won't work by default if set to false. Webhook needs certs for working